src/app/helpers/telemetryHelper.js

```diff
@@ -529,5 +529,23 @@ module.exports = {
       type: 'UserSession'
     });
     return cdata;
+  },
+
+/**
+ * This function used to generate api_access log event
+ */
+  getTelemetryAPISuceessData: function(result, req, uri) {
```

Comment:
We dont need new method here. We can use the existing log method itself by passing right EDATA.

Reply:
in this method, I m preparing the req data for the existing function.